assets/js/analysis/new.controller.js

```diff
@@ -4,8 +4,9 @@
   .module('linguine.analysis')
   .controller('AnalysisNewController', AnalysisNewController);
 
-  function AnalysisNewController($http, $scope, $state, $rootScope, flash) {
+  function AnalysisNewController($http, $scope, $state, $rootScope, flash, usSpinnerService) {
     $scope.analysisNotSelected = true;
+    $scope.needTokenizer = true; 
```

Comment:
Maybe change the name of this to `tokenizerSelected`.  The way it's named now makes me think it marks whether or not a tokenizer is necessary for a given analysis.